x-pack/legacy/plugins/uptime/public/components/functional/empty_state/empty_state_error.tsx

```diff
@@ -13,25 +13,45 @@ interface EmptyStateErrorProps {
   errors: GraphQLError[];
 }
 
-export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => (
-  <EuiPanel>
-    <EuiEmptyPrompt
-      title={
-        <EuiTitle size="l">
-          <h3>
-            {i18n.translate('xpack.uptime.emptyStateError.title', {
-              defaultMessage: 'Error',
-            })}
-          </h3>
-        </EuiTitle>
-      }
-      body={
-        <Fragment>
-          {errors.map((error: GraphQLError) => (
-            <p key={error.message}>{error.message}</p>
-          ))}
-        </Fragment>
-      }
-    />
-  </EuiPanel>
-);
+export const EmptyStateError = ({ errors }: EmptyStateErrorProps) => {
+  const unAuthorized = errors.find(
+    (error: GraphQLError) => error.message && error.message.includes('unauthorized')
+  );
+
+  return (
+    <EuiFlexGroup justifyContent="center">
+      <EuiFlexItem grow={false}>
+        <EuiPanel>
+          <EuiEmptyPrompt
+            iconType="securityApp"
+            iconColor="subdued"
+            title={
+              <EuiTitle size="m">
+                {unAuthorized ? (
+                  <h3>
+                    {i18n.translate('xpack.uptime.emptyStateError.noPermission', {
```

Comment:
```suggestion
                    {i18n.translate('xpack.uptime.emptyStateError.notAuthorized', {
```

This is mostly just a suggestion but it's closer to the jargon that describes what's actually happening to trigger this state.